sites/svelte.dev/src/routes/_components/Try.svelte

```diff
@@ -0,0 +1,121 @@
+<script>
+	import { Section } from '@sveltejs/site-kit/components';
+	import { TryTerminal } from '@sveltejs/site-kit/components';
+</script>
+
+<div class="try-container">
+	<Section --background="var(--background-2)">
+		<div class="grid" style="--columns: 2">
+			<div class="try">
+				<TryTerminal />
+
+				<p class="create-an-app">
+					...or
+					<a target="_blank" rel="noreferrer" href="https://sveltekit.new"> create an app </a>
+					on StackBlitz.
+				</p>
+			</div>
+
+			<div class="description">
+				<p>
+					Svelte is a radical new approach to building user interfaces. Whereas traditional
+					frameworks like React and Vue do the bulk of their work in the <em>browser</em>, Svelte
+					shifts that work into a
+					<em>compile step</em> that happens when you build your app.
+				</p>
+
+				<p>
+					Instead of using techniques like virtual DOM diffing, Svelte writes code that surgically
+					updates the DOM when the state of your app changes.
+				</p>
```

Comment:
this feels a bit text-heavy. let's make it a bit more concise
```suggestion
				<p>
					Svelte is a radical new approach to building user interfaces. Whereas traditional
					frameworks like React and Vue do the bulk of their work in the <em>browser</em>, Svelte
					shifts that work into a
					<em>compile step</em> that happens when you build your app. No more virtual DOM!
				</p>
```